README.md

```diff
@@ -79,6 +79,25 @@ You only need to specify in what language is wrote the text that you're passing
 
 Also, additionally you can specify the path of the configuration file, by default it will search for the file called `i18n-auto-translate.config.json` on the root of your project, but you can specify another path with the `-s` or `--settings-file` flag.
 
+### Nested Translations
```

Comment:
Maybe also add the same or similar examples as in dset:

https://github.com/lukeed/dset#usage

To help users identify what can be done and what not

Reply:
Added some new examples with what can be done, please take a look when u can